website/docs/r/lb_listener_rule.html.markdown

```diff
@@ -92,8 +102,16 @@ resource "aws_lb_listener_rule" "health_check" {
   }
 
   condition {
-    field  = "path-pattern"
-    values = ["/health"]
+    field  = "query-string"
+
+    query_string {
+      values = [
+        {
+          key   = "health"
+          value = "check"
+        },
+      ]
```

Comment:
What about switching to:
```suggestion
query_string {
    values {
        key   = "health"
        value = "check"
    }
}
```

Seems more readable, thoughts?

Reply:
See my `query-string` comments above. This syntax is unfortunately verbose as it follows the API pattern shared by all the new blocks. It could be reduced and become more unique.

Reply:
You're quite right. That syntax works. I've updated the example.